### PR TITLE
Mahesh_Assignment_29_15mins_Done

### PR DESCRIPTION
**Pull request by MaheshKJun20**
Mahesh_Assignment_29_15mins_Done

### CODE REVIEW

**Comment by MaheshKJun20** [comment]
image

**Review by harshvegada — APPROVED** [review]
Good @MaheshKJun20 you wrote two function but always go with approach which is done by exiting methods, like you use methods on Character class, if interviewer ask you different approach which not using any exiting method then go for ASCII logic.

both logic is correct
keep it up!

File: src/maheshKJun20/StringExamples/ConvertCase.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+		return newStr;
+	}
+	
+	//Optional Method to convert case of each character in String and return the converted string value. It is using Ascii value of character.
+	String convertCaseInStringUsingAscii(String str) {
+		String newStr = "";
+		for (int index=0;index<str.length();index++) {
+			int asciiValue = str.charAt(index);
+			if (asciiValue >= 65 && asciiValue <= 90)
+				newStr += Character.toLowerCase(str.charAt(index));
```
Comment (KrishnaKTechnocredits) [inline src/maheshKJun20/StringExamples/ConvertCase.java:46]: If interviewer will ask you not to use  toLowerCase/toLowerCase method then, what you will do ?

Comment (MaheshKJun20) [reply src/maheshKJun20/StringExamples/ConvertCase.java:46]: Hi Krishna, 
We can use Ascii value to convert case
example - asciivalue +/- 32 
will udpate the code with this logic in second function to make it purely ascii based logic.
Thanks

**Pull request merged**